normalizer_test.go

```diff
@@ -0,0 +1,500 @@
+package sqllexer
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestNormalizer(t *testing.T) {
+	tests := []struct {
+		input          string
+		want           string
```

Comment:
I'd name this `expected` instead since that's the usual name used by popular packages like testify.
```suggestion
		expected           string
```